Content.Shared/GameObjects/EntitySystems/SharedPullingSystem.cs

```diff
@@ -67,6 +70,28 @@ private void PullerMoved(MoveEvent ev)
             physics.WakeBody();
         }
 
+        // TODO: When Joint networking is less shitcodey fix this to use a dedicated joints message.
+        private void HandleContainerInsert(EntInsertedIntoContainerMessage message)
+        {
+            if (message.Entity.TryGetComponent(out SharedPullableComponent? pullable))
+            {
+                pullable.TryStopPull();
+                return;
```

Comment:
should this really return? if something is pullable its not garantueed to not be a puller right?

Reply:
You're right I'm stupid and forgot chaining.